packages/playground/src/portal/transfer_view.vue

```diff
@@ -0,0 +1,241 @@
+                  validators.isAlphanumeric('Invalid Address'),
+                  isSameAddress,
+                ]"
+                :async-rules="[isValidAddress]"
+                #="{ props }"
+              >
+                <input-tooltip tooltip="Enter Address of Receipient Account">
+                  <v-text-field label="Recipient Address:" v-model="receipientAddress" v-bind="props"> </v-text-field>
+                </input-tooltip>
+              </input-validator>
+              <input-validator
+                v-if="!loadingBalance"
+                :value="transferAmount"
+                :rules="[
+                  validators.required('Transfer amount is required '),
+                  validators.isNumeric('Amount should be a number.'),
+                  validators.min('Amount must be greater than 0', 0.00000000001),
+                  validators.max('Insuffient funds', freeBalance),
+                ]"
+                #="{ props }"
+              >
+                <input-tooltip tooltip="0.01 transaction fee will be deducted">
+                  <v-text-field label="Transfer Amount:" v-bind="props" v-model.number="transferAmount"></v-text-field>
+                </input-tooltip>
+              </input-validator>
+              <strong v-else>Loading...</strong>
+            </form-validator>
+            <v-card-actions>
+              <v-spacer></v-spacer>
+              <v-btn @click="clearInput" class="primary white--text">Clear</v-btn>
+
+              <v-btn
+                class="primary white--text"
+                :loading="loadingAddressTransfer"
+                :disabled="!isValidAddressTransfer"
+                @click="submitFormAddress"
+                color="secondary"
+                variant="tonal"
+                >Send</v-btn
+              >
+            </v-card-actions>
+          </v-card>
+        </v-window-item>
+      </v-window>
+    </v-card>
+  </v-container>
+</template>
+<script lang="ts" setup>
+import { Keyring } from "@polkadot/keyring";
+import { GridClient } from "@threefold/grid_client";
+import type { Twin } from "@threefold/tfchain_client";
+import { createToast } from "mosha-vue-toastify";
+import { onMounted, ref } from "vue";
+
+import { useProfileManager } from "../stores";
+
+const activeTab = ref(0);
+const receipientTwinId = ref("");
+const isValidTwinIDTransfer = ref(false);
+const transferAmount = ref(1);
+const loadingTwinIDTransfer = ref(false);
+const loadingAddressTransfer = ref(false);
+const isValidAddressTransfer = ref(false);
+const receipientAddress = ref("");
+const profile = useProfileManager().profile;
+const loadingBalance = ref(true);
+const recepTwinFromAddress = ref<Twin>();
+const receptTwinFromTwinID = ref<Twin>();
+const gridClient = new GridClient({
+  mnemonic: useProfileManager().profile!.mnemonic,
+  network: window.env.NETWORK,
+});
+const freeBalance = ref(0);
+onMounted(async () => {
+  await gridClient.connect();
+  await getFreeBalance();
+});
```

Comment:
Please use `getGrid` util as it injects all urls which is important is custom networks (might be added later)